### PR TITLE
Patch to fix some warnings, renamed rm_CAN file to rm_can

### PR DESCRIPTION
**Pull request by Pandabear1125**

### CODE REVIEW

**Comment by Pandabear1125** [comment]
Still a warning in dr16 however. I cant really remove it since it's technically a 'used' variable but it is not currently used.

**Comment by Pandabear1125** [comment]
This PR has kinda morphed into "do any tidying up".

File: src/comms/rm_can.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@ void rm_CAN::read() {
     CAN_message_t msg1;
     uint16_t read1;
 
-    while (read1 = m_can1.read(msg1)) {
+    while ((read1 = m_can1.read(msg1))) {
```
Comment (nitroxplunge) [inline src/comms/rm_can.cpp:25]: Redundant parentheses can be really confusing

File: src/comms/rm_can.cpp [hunk 2]
```diff
@@ -36,7 +36,7 @@ void rm_CAN::read() {
     CAN_message_t msg2;
     uint16_t read2;
     
-    while (read2 = m_can2.read(msg2)) {
+    while ((read2 = m_can2.read(msg2))) {
```
Comment (nitroxplunge) [inline src/comms/rm_can.cpp:39]: Same here

**Review by Rishi-H — APPROVED** [review]

**Review by nitroxplunge — APPROVED** [review]
Approved, but I suggest removing the redundant parentheses

**Pull request merged**